### ISSUE TITLE
Styling issues with Lorenz notebook differ from Jupyter Notebook

### ISSUE THREAD

**Issue by jhpedemonte** [jupyter-incubator/dashboards_server#147]
Found a couple of styling issues when running **demo/lorenz.ipynb**:

1. I enabled showing the 2nd cell which has some equations. The equations render correctly in the Notebook, but not on the Dashboard Server. I'm guessing this requires a JS library to read this particular markup.

2. In the Notebook, the sliders are all the same width and take up a good amount of space. In the Dashboard Server, the sliders are short and variable width.

**Comment by jhpedemonte** [comment]
#### Notebook:

![screen shot 2016-03-18 at 11 35 25 am](https://cloud.githubusercontent.com/assets/711365/13884827/b8f1f3f8-ecfd-11e5-888d-29c105eba003.png)

#### Dashboard Server:

![screen shot 2016-03-18 at 11 35 40 am](https://cloud.githubusercontent.com/assets/711365/13884834/bfc9f6d0-ecfd-11e5-88ef-aae2cd260dc9.png)

**Comment by parente** [comment]
> I'm guessing this requires a JS library to read this particular markup.

Equations require pulling in mathjax. Unfortunately, mathjax is provided by the jupyter notebook app environment rather than directly imported by the notebook that will use the text. We'll have to do the same here if we want to match.

**Comment by parente** [comment]
The sizing of the sliders and whatnot are fixed by moving to ipywidgets 5.0.0 in #194 but the labels and default values have disappeared.

**Comment by parente** [comment]
Punt on mathjax for now. We'll track that separately since it's another can of worms. Just get the sizing right on the sliders.

**Comment by jhpedemonte** [comment]
So sizing of sliders is hard-coded in *jupyter-js-widgets* to 300px, whereas in the Notebook it's greater. Tried to run lorenz (or just even a simple slider) in JupyterLab, but that wasn't working. Still have not been able to run an actual notebook in JupyterLab!

We may just have to live with these differences. Eventually, when everything is using *jupyter-js-widgets*, we'll look the same.

**Comment by parente** [comment]
Confirmed. Here's notebook 4.2 with jupyter-js-widgets 5.1.1:

![screen shot 2016-04-28 at 4 28 08 pm](https://cloud.githubusercontent.com/assets/153745/14900392/50897be8-0d5e-11e6-86b6-bb1b44a317be.png)

Looks like dashboard server.

So no sense in trying to match to the older version of notebook. We'll just need to float along and match whatever the current lib version give us.
